packages/cli/test/unit/PostHog.test.ts

```diff
@@ -21,17 +25,17 @@ describe('PostHog', () => {
 	});
 
 	it('inits PostHog correctly', async () => {
-		const ph = new PostHogClient();
-		await ph.init(instanceId);
+		const ph = new PostHogClient(instanceSettings);
```

Comment:
Curious, if all this client needs is the `instanceId`, should we just pass only that in? Or does it make no difference?

Reply:
we could pass in only the `instanceId`, but this is an injected service now, so the `instanceId` would need to be wrapped in a typedi `Token`. It's less code to inject `InstanceSettings` instead.

Reply:
I'll create another PR to cleanup how `PosthogClient` is setup, so that we can also stop optionally passing it in controllers.